bin/cv_regress

```diff
@@ -173,6 +168,16 @@ def read_file(args, file):
         elif test.num != 1:
             test.num = int(args.num or test.num)
 
+        try:
```

Comment:
I can see the utility of skipping a specific simulation for a given configuration, but it is not clear to me why we would skip a simulaton based on the simulator used.  Having said that, this `test.simulator` check has been in this script for a long time, so I will not gate approval of this PR on it.

Nevertheless, I am curious - @silabs-hfegran can you comment?

Reply:
I was not aware of this code in the cv_regress script.

That said, the only thing that comes to mind is if a specific test triggers a known issue with a certain simulator - I guess this was more relevant back when we had multiple tools running the various testbenches to avoid having to do a lot of unrelated maintenance.